fbw-common/src/wasm/systems/systems/src/shared/mod.rs

```diff
@@ -815,6 +817,71 @@ impl From<MachNumber> for f64 {
     }
 }
 
+impl MaxDifference for MachNumber {
+    fn max_diff(self, other: Self) -> f64 {
+        (f64::from(self) - f64::from(other)).abs()
+    }
+}
+
+impl MachNumber {
+    // All formulas from Jet Transport Performance Methods by Boeing (March 2009 revision)
+
+    /// Get the pressure ratio for a given pressure
+    fn delta(air_pressure: Pressure) -> f64 {
```

Comment:
In my opinion this function should be called `pressure_ratio` because delta normally means something else.
Also I would change the return value to `Ratio`.

Reply:
Delta fits with the formulas it is used in (Greek letter delta).